MAiD - Case Management App/force-app/main/default/objects/Referral__c/validationRules/Phone_number_Validation.validationRule-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Phone_number_Validation</fullName>
+    <active>true</active>
+    <errorConditionFormula>OR(AND(Individual_Cell_Phone_Number__c!=null,
+NOT(REGEX(Individual_Cell_Phone_Number__c,&quot;(\\D?[0-9]{3}\\D?)[\\s][0-9]{3}-[0-9]{4}&quot;)),NOT(REGEX(Individual_Cell_Phone_Number__c,&quot;^[0-9]{10}$&quot;))),
```

Comment:
Review and testing indicates that an additional phone number is present in the interface and needs to be added: Individual_Home_Phone_Number__c @lilianenassar-bib Thank you.